### DISCUSSION TITLE
Documentation about manual installation of the add-on

### DISCUSSION THREAD

**Discussion by tomaso** [Koenkk/zigbee2mqtt#16292] [General]
Hi,
I have been running HA in a docker but I don't use supervisor nor the HA OS - just the plain docker with ghcr.io/home-assistant/home-assistant image and I manage other container manually (e.g. koenkk/zigbee2mqtt). 
I tried to search the documentation of the zigbee2mqtt HA add-on but I found only instructions (linked from https://www.zigbee2mqtt.io/guide/installation/03_ha_addon.html) about installing the add-in via HA store and HA UI.
Is there any info on how to install the add-on manually? I checked generic HA documentation on add-ons but it says that I should check add-on specific docs.
Thank you for any help.
Cheers,
Tomas

**Comment by Koenkk** [comment]
AFAIK you cannot do this. The HA addon requires the HA supervisor which is missing.

**Reply by tomaso** [reply]
Thanks for your reply. In that case is it possible to even use zigbee2mgtt in HA without the add on?

**Reply by Nardol** [reply]
I am not sure to understand what is wrong for you.
If you install Zigbee2MQTT following [the documentation to install using docker](https://www.zigbee2mqtt.io/guide/installation/02_docker.html), have a running MQTT broker and configured both Home Assistant and Zigbee2MQTT to point to the same broker it should work.
Then you could read the documentation about how to [integrate ZB2MQTT to HA](https://www.zigbee2mqtt.io/guide/configuration/homeassistant.html) to be sure all is OK to auto-discover Zigbee devices in Home Assistant.
This last link also point to [the Home Assistant integration guide](https://www.zigbee2mqtt.io/guide/usage/integrations/home_assistant.html).

The addon is only needed for Home Assistant OS and supervised installation.

**Reply by tomaso** [reply]
@Nardol thanks a lot. The second link was the part I was missing. My understanding was that I need to configure more on HA side. I didn't get that it is actually the zb2mqtt that will export data in the way that HA understands. I can see my devices in HA now. Thanks again :)
